app/assets/javascripts/components/subjects_taught.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
+  var fieldset = document.querySelector("#tslr_claim_eligible_subjects");
+  var el;
+  var eligibleSubjectsSelector;
+  var notTeachingSubjectsSelector;
```

Comment:
No need to declare these outside of the function scope since we're not reusing their values.